src/components/icons/svg/SatoshiIconCircled.tsx

```diff
@@ -0,0 +1,41 @@
+import { Icon, IconProps } from '@chakra-ui/react'
+
+export interface SatoshiIconCircledProps extends IconProps {
+  size?: 'lg' | 'md' | 'sm'
+}
```

Comment:
You're using this in a few places, maybe standardize this. 
You can check it these are the sizes in the design system with vlad, and if so. you can add these to the Icon theme for chakra. 

If not, a normal defined type should be okay too.